validator/js/chromeextension/build_extension.sh

```diff
@@ -2,6 +2,8 @@
 #
 # A script to install dependencies and then build the extension.
 
+curl -f https://cdn.ampproject.org/v0/validator_wasm.js -o validator_wasm.js
```

Comment:
when does `build_extension.sh` get called btw?

Reply:
Everything is manual as of now. Need to call this every time to manually build the extension.